templates/cpp-template-default/proj.android/app/build.gradle

```diff
@@ -111,19 +111,22 @@ android {
 }
 
 android.applicationVariants.all { variant ->
-    def project_root_folder = "${buildDir}/../../.."
-    def dest_assets_folder = "${buildDir}/../assets"
+    def project_root_folder = "${projectDir}/../.."
+    def dest_assets_folder = "${projectDir}/assets"
```

Comment:
What's the value of `projectDir `?

Reply:
According https://docs.gradle.org/current/userguide/writing_build_scripts.html

> The directory containing the build script.